modules/goods/server/models/good.server.model.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/**
+ * Module dependencies
+ */
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+/**
+ * Good Schema
+ */
+var GoodSchema = new Schema({
+  name: {
+    type: String,
+    default: '',
+    trim: true,
+    required: 'Name cannot be blank'
+  },
+  description: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  price: {
+    type: Number,
+    required: 'Price cannot be blank'
+  },
+  donor: {
+    firstName: {
+      type: String,
+      trim: true,
+      default: '',
+      required: 'Donor\'s first name cannot be blank'
+    },
+    lastName: {
+      type: String,
+      trim: true,
+      default: '',
+      required: 'Donor\'s last name cannot be blank'
+    },
+    email: {
+      type: String,
+      trim: true,
+      default: ''
+    },
+    receiptDate: {
+      type: Date
```

Comment:
In my local testing, I wrote a test to see if I can save an empty `receiptDate` and it passed. I dont know if that is a test we need to add. I tried that out because I did not know how mongodb does `Date` fields and can it accept empty `Dates`